index.html

```diff
@@ -0,0 +1,202 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+        rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+            rel="stylesheet">
+</head>
+
+<body>
+    
+    <!----------navbar setion------------------->
+    <header>
+    <nav class="nav-container">
+        <a href="">John Doe</a>
+        <ul class="items-menu">
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+            <li><i class="fa-solid fa-envelope"></i></li>
+        </ul>
+        <a href="#"><i class="fa-solid fa-bars"></i></a>
+    </nav>
+    </header>
+    
+    <!--------------Hero section------------------------------------------->
+    <section>
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <ul>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+                </ul>
+            </div>
+            <div class="hero-right">
+                <h1>Hi There.<br>I am Hyder</h1>
+                <h5>I'm a web Developer</h5>
+                <p>I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you
+                    see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div class="hero-right1">
+                
+            </div>
+        </div>
+
+    </div>
+    </section>
+    <!------------------myworks---------------------------------------->
+    <div class="myworks">
+        <div class="work-heading">
+            <h4>My Recent Works</h4>
+            <hr class="hline">
+        </div>
+        <div class="work-details">
+            <div class="work-details-row1">
+                <img src="./images/Img Placeholder (1).png">
```

Comment:
- [ ] You did a wonderful job inserting the image using `<img>` tag :+1: :100:, but I have noticed that there are no images on the cards as you can see below, so please don't forget to add the required images :+1: 

![Screenshot from 2023-01-17 05-01-57](https://user-images.githubusercontent.com/105117832/212868533-a411b262-bb12-4465-b9fb-8f8630c98575.png)